go/obscuronode/nodecommon/rpc/generated/README.md

```diff
@@ -5,6 +5,7 @@ Install Protobuf and Protoc-gen-go with:
 
     brew install protobuf
 
+    go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
```

Comment:
Does this install protobuff twice ? As in, do we need brew too ?

Reply:
For me, I'd done the brew install and the protoc-gen-go-grpc as per this readme but I couldn't get the generation to work until I installed this thing too. Maybe that's something specific to my setup though, not sure...

Did these instructions work for you as they were before?

Reply:
Yeah it works for me, can you check that you don't have 2 installs of protobuff ?
I'm spitballing, but I'd guess it's a $PATH issue 🤔

Reply:
I don't think so, can just see the homebrew one, where would the other be?

This guide suggests you need to install protoc and also the two go plugins for it: https://grpc.io/docs/languages/go/quickstart/